src/burnchains/burnchain.rs

```diff
@@ -637,11 +640,14 @@ impl Burnchain {
     }
 
     /// Try to parse a burnchain transaction into a Blockstack operation
+    /// `pre_stx_op_map` should contain any valid PreStxOps that occurred before
+    ///   the currently-being-evaluated tx.
     pub fn classify_transaction(
         burnchain_db: &BurnchainDB,
         block_header: &BurnchainBlockHeader,
         burn_tx: &BurnchainTransaction,
         sunset_end_ht: u64,
+        pre_stx_op_map: &HashMap<Txid, PreStxOp>,
```

Comment:
These are only the ones that occur in the same block though, right?

Reply:
Yep -- I'll be more specific in the rustdoc description.

Reply:
Addressed in https://github.com/blockstack/stacks-blockchain/pull/2129/commits/b4c7b8878ad442a81a5fe47b6e94de287de49305